src/redux/reducers/instructorReducer.js

```diff
@@ -32,6 +32,7 @@ const initialState = {
   post: [],
   postOptions: 'newsFeed',
   postID: 0,
+  posted_at: null,
```

Comment:
only hesitation I have with a 'null' case here is that your API _may not_ be set up to receive a null in the instance where this ID could be type `str` or `int/Number`